### ISSUE TITLE
Clicking object does not popup inspector view on Safari

### ISSUE THREAD

**Issue by pocka** [pocka/figspec#37]
https://github.com/pocka/figspec/pull/36#issue-1174131996

Confirmed on macOS Safari Version 15.1 at [4996f1c](https://github.com/pocka/figspec/pull/12/commits/4996f1cba41849ea2fbb2ca8463069e07e76f5c0) and `master`.

**Comment by pocka** [comment]
https://stackoverflow.com/questions/42938270/click-event-on-svg-not-working-in-safari

**Comment by e111077** [comment]
some investigation:

* seems to be broken in the storybook since before lit upgrade
* I cannot reproduce this using figspec/components outside of this local storybook ([e.g.](https://lit.dev/playground/?mods=gists#gist=d870e2ddb1e8dbb30d14fa3716685c8d))
* Messing around with making the `<path>` of the node outlines not overflow of the `<svg>` in the storybook demo in this repo, it functions normally

Basically: Pretty sure you nailed down the issue, but I don't know why it's broken for this storybook but not for other demos that are in a similar situation

**Comment by pocka** [comment]
@e111077 
Thanks for the insight! The second point, that the selection does work outside Storybook, is interesting.

My current plan is to replace the SVG outlines with Canvas, as (probably) SVG rendering is the cause of #30 and #31.

**Comment by e111077** [comment]
Moving to canvas was good enough for Google Docs, it's good enough for figspec!

Not much you can do to get around the giant JSON binary in memory.

Perhaps mouse interactions with canvas can stream to a WASM binary that preprocesses the JSON which streams back rendering data? Though I wonder how slow the communication across the binary would be. A lot of work but sounds like fun!

**Comment by pocka** [comment]
[v1.0.2](https://github.com/pocka/figspec/releases/tag/v1.0.2)
